app/src/main/java/org/simple/clinic/selectcountry/SelectCountryModel.kt

```diff
@@ -1,10 +1,11 @@
 package org.simple.clinic.selectcountry
 
 import org.simple.clinic.appconfig.Country
+import org.simple.clinic.util.ResolvedError
 
 data class SelectCountryModel(
     val countries: List<Country>?,
-    val manifestFetchError: ManifestFetchError?,
+    val manifestFetchError: ResolvedError?,
```

Comment:
Can we model these as domain errors instead? (Details discussed offline).

Reply:
Done.